week-4/implement/get-ordinal-number.test.js

```diff
@@ -1,4 +1,18 @@
 // In this week's prep, we started implementing getOrdinalNumber
+function getOrdinalNumber(num) {
+  if (num % 10 === 1) {
+    if (num === 11) {
+      return "11th";
```

Comment:
I did mine a bit differently. Mine is a bit more concise, but this makes sense as well. Well done!